pkg/pillar/cmd/logmanager/logmanager.go

```diff
@@ -404,8 +410,18 @@ func parseAndSendSyslogEntries(ctx *loggerContext) {
 		}
 		level, err := log.ParseLevel(logInfo.Level)
 		if err != nil {
-			log.Errorf("ParseLevel failed: %s\n", err)
-			level = log.InfoLevel
+			// XXX Some of the log sources send logs with
+			// severity set to err, emerg & notice.
+			// Logrus log level parse does not recognize the above severities.
+			// Map err, emerg to error and notice to info.
+			if logInfo.Level == "err" || logInfo.Level == "emerg" {
+				level = log.ErrorLevel
+			} else if logInfo.Level == "notice" {
+				level = log.InfoLevel
+			} else {
```

Comment:
Wouldn't it make sense to add a local function parseLogLevel() which does this?
Also, just in case, should we downcase logInfo.Level before calling ParseLevel and comparing it with "notice" etc

Reply:
Done. Committed my changes.